$OpenBSD: patch-vmime_address_hpp,v 1.1 2017/05/04 18:01:19 espie Exp $
recent C++ needs this to be public, so does -current libvmine.

Index: vmime/address.hpp
--- vmime/address.hpp.orig
+++ vmime/address.hpp
@@ -65,8 +65,6 @@ class address : public headerFieldValue (public)
 
 	virtual ref <component> clone() const = 0;
 
-protected:
-
 	/** Parse an address from an input buffer.
 	  *
 	  * @param buffer input buffer
